src/assets/css/style.css

```diff
@@ -0,0 +1,67 @@
+.wp-block-columns {
```

Comment:
Since we are are using material ui what do you think about placing this in the Theme.js file. 

```
override: {
MuiCssBaseline: {
'@global': {
'.wp-block-columns': {
...
}
...
}
}
}
```

cc @kilemensi

Reply:
@karimkawambwa :+1: